mlir/test/mlir-miopen-driver/populate_subkernels.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: mlir-miopen-driver -conv-config "--operation conv2d --arch gfx906 --num_cu 64 --fil_layout GNCHW --in_type fp32 --fil_type fp32 --out_type fp32 --in_layout NGCHW --out_layout NGCHW --batchsize 64 --in_channels 1024 --out_channels 1024 --in_h 14 --in_w 14 --out_h 14 --out_w 14 --fil_h 1 --fil_w 1 --dilation_h 1 --dilation_w 1 --conv_stride_h 1 --conv_stride_w 1 --padding_h 0 --padding_w 0 --kernel_name conv2d_fwd --groupsize 1 --kernel_id 1" | FileCheck %s --check-prefix=KERNEL1
```

Comment:
how about add one more line with `kernel_id 0`?

and perhaps add some comments in this file to remind ourselves that all of these shall be changed when `miopen.conv2d_bwd_data` implementation is ready.